azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/CallCompositeException.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling;
+
+import androidx.annotation.NonNull;
+
+/**
+ * Defines the base type of custom Exception that can be thrown by this Library.
+ */
+public final class CallCompositeException extends RuntimeException {
```

Comment:
We currently have more than one class with the same signature for similar purpose in the composite. Is it possible to rename one of those classes and reuse them instead of having a new exception class? Or using an instance of this in error handling purpose? Whichever suits better.